shared/tests/CMakeLists.txt

```diff
@@ -2,15 +2,15 @@
 file(GLOB_RECURSE test_src "*/**.cpp" "*/**.hpp")
 
 if(test_src)
-  add_executable(TestShared "${CMAKE_SOURCE_DIR}/${NUCLEAR_SHARED_DIR}/tests/test.cpp" ${test_src})
-  target_link_libraries(TestShared PRIVATE nuclear::message nuclear::utility)
+  add_executable(testshared "${CMAKE_SOURCE_DIR}/${NUCLEAR_SHARED_DIR}/tests/test.cpp" ${test_src})
+  target_link_libraries(testshared PRIVATE nuclear::message nuclear::utility)
```

Comment:
What do all these changes to lowercase `testshared` achieve?

Reply:
a binary named `testshared`. The advantage of having lower case names is that we can `to_lower()` string inputs people give to run specific tests, which means they can be case insensitive.